scripts/examples-checker/Cargo.toml

```diff
@@ -8,3 +8,5 @@ publish = false
 [dependencies]
 anyhow = "1.0.41"
 clap = { version = "3.1", features = ["env", "derive"] }
+forc-util = { version = "0.12.2", path = "../../forc-util" }
+tracing = "0.1"
```

Comment:
Similarly to the `mdbook-forc-documenter`, as this is an internal script purely for CI and local testing, I think we can probably leave out `tracing` from this crate and stick to `println!`.